src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -356,5 +359,20 @@ Polymer({
     const port = Number(value);
     const valid = !Number.isNaN(port) && port >= 1 && port <= 65535 && Number.isInteger(port);
     return valid ? '' : this.localize('error-keys-port-bad-input');
+  },
+
+  _getInternationalizedUnit(bytesAmount, language) {
+    // This happens during app startup before we set the language
```

Comment:
I've run into these kind of issues a few times too.

My approach has been to try to set a sensible default and always maintain an invariant that the value is not null. That allowed me to remove the `null|undefined` checks.

What are your thoughts?

Reply:
That seems like a good default to me. I suspect the user will not even see the default value since it's not the default tab.